test cases/fortran/9 cpp/fortran.f90

```diff
@@ -0,0 +1,11 @@
+function fortran() bind(C)
```

Comment:
This test file had the suffix `.f` specifically to test that files with such extension get compiled. Are there compilers that do not handle `.f` files or why was this renamed?